ignite/cmd/network_chain_prepare.go

```diff
@@ -28,6 +29,7 @@ func NewNetworkChainPrepare() *cobra.Command {
 	}
 
 	c.Flags().BoolP(flagForce, "f", false, "Force the prepare command to run even if the chain is not launched")
+	c.Flags().String(flagChainID, cosmosutil.SPNChainID, "Chain ID to use for this network")
```

Comment:
For which network, SPN?

Maybe we update the description add change the flag to `--spn-chain-id`. Otherwise, people may think that it's their chain's id.

Reply:
I agree. But maybe we need to start avoid the `spn` and start to use `network` or `blockchain.`

Reply:
It can be done in a separate PR for all, or now it's OK to use spn.